robot-custom-integration/sl_python_robot/SLListener.py

```diff
@@ -42,19 +45,22 @@ def start_suite(self, suite, result):
                 test.body.create_keyword(name="SKIP")
 
         tests_for_execution = list(all_tests - self.excluded_tests)
-        print(f'>>>> Tests for execution: {tests_for_execution}')
+        print(f'{SEALIGHTS_LOG_TAG} {len(tests_for_execution)} Tests for execution: {tests_for_execution}')
```

Comment:
word "Tests" should not be capitalized as it's not the first one in the sentence anymore.